scripts/server.js

```diff
@@ -717,11 +717,11 @@ setups.getBackendSetups(function (setups) {
 
   // Attach any tool plugins to the application
   // We're only doing this for dev because we're assuming
-  // build will also call this.
-  if (process.env.NODE_ENV === "development") {
-    console.log(chalk.cyan("Updating Tools...\n"));
-    updateTools();
-  }
+  // // build will also call this.
+  // if (process.env.NODE_ENV === "development") {
+  //   console.log(chalk.cyan("Updating Tools...\n"));
+  //   updateTools();

```

Comment:
This is in fact needed. MMGIS can link into sub-repos of private tools and this `updateTools` functions scans for them and plugs them in.

Reply:
I offer my sincerest apologies for the error that occurred in my recent upload. My intention was to upload changes to my fork specifically, but by mistake, I uploaded them to the main repository instead. These changes were designed to simplify the process of running the project in development mode on my personal device, as I am not currently part of the NASA team. Regrettably, I also submitted a pull request to the master branch instead of creating a new branch named "spaceapps."

Upon reflection, I now understand that I should have exercised greater caution with respect to the timing of my pull request. I had previously created a new branch and made the necessary alterations in separate commits. However, during the pull request process, I became confused and made an error. I am deeply sorry for any inconvenience or confusion my actions may have caused.

I assure you that this was an unfortunate mistake, and I am committed to ensuring that it will not occur in the future. Once again, I apologize for any inconvenience caused.

Reply:
No worries whatsoever! This is what PRs are for too.